helm-charts/templates/data-plane/gateway-components/log-conf.yaml

```diff
@@ -109,8 +109,14 @@ data:
         connectionTimeout = "{{ .Values.wso2.apk.dp.gatewayRuntime.tracing.configProperties.connectionTimeout }}"
         {{end}}
     {{end}}
+
+    {{- if .Values.wso2.apk.dp.gatewayRuntime.analytics }}
```

Comment:
shall we generate helm chart readme

Reply:
Updated the readme by https://github.com/wso2/apk/pull/1224/commits/c5691c758474f0bdca0d06358745471755e5a1bf.